arm_robots/scripts/tutorial/estimated_torques_demo.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+import argparse
+import sys
+
+import rospy
+from arc_utilities import ros_init
+from arm_robots.get_robot import get_moveit_robot
+from geometry_msgs.msg import Wrench
+from moveit_msgs.msg import RobotState
+from sensor_msgs.msg import JointState
+
+torque_viz_scales = {
+    "joint56": 500.0,
+    "joint57": 50.0,
+    "joint1":  10.0,
+    "joint2":  40.0,
+    "joint3":  100.0,
+    "joint4":  100.0,
+    "joint5":  100.0,
+    "joint6":  1000.0,
+    "joint7":  1000.0,
+    "joint41": 10.0,
```

Comment:
What are joints 56,57 and 41-47 (other arm?) Maybe some comments here?

Reply:
Oh I see this is for val

Reply:
I've added a print statement so it says which robot it's using. But yes this script currently only works for val. Seems less likely someone will want it for victor because victor actually has torque sensors